apps/erlangbridge/src/vscode_lsp_entry.erl

```diff
@@ -30,8 +30,8 @@ compile_needed_modules() ->
     do_compile(["src/vscode_lsp_app", "src/gen_lsp_server", 
         "src/gen_lsp_sup", "src/gen_lsp_doc_sup","src/gen_lsp_doc_server", "src/gen_lsp_config_sup","src/gen_lsp_config_server",
         "src/gen_lsp_help_sup","src/gen_lsp_help_server", "src/lsp_handlers", "src/lsp_utils",
-        "src/vscode_lsp_app_sup", "src/lsp_navigation", "src/lsp_syntax", "src/lsp_completion",
-        "src/gen_connection", "src/vscode_jsone","src/vscode_jsone_decode","src/hover_doc_layout"], CompileOptions)
+        "src/vscode_lsp_app_sup", "src/lsp_navigation", "src/lsp_parse", "src/lsp_syntax", "src/lsp_completion",
+        "src/gen_connection", "src/vscode_jsone","src/vscode_jsone_decode","src/hover_doc_layout", "src/worker.erl"], CompileOptions)
```

Comment:
to be homogeneous, with out ".erl" for worker.erl

Reply:
Oops